src/pykeen/trackers/__init__.py

```diff
@@ -31,5 +31,6 @@
 
 tracker_resolver = Resolver.from_subclasses(
     base=ResultTracker,
+    default=ResultTracker,
```

Comment:
i was wondering if that made sense, but technically the ResultTracker isn't even an abstract class, but sort of a no-op interace

Reply:
hm, we could also add another NoOp tracker class, but I am okay with how it is now.

Reply:
I agree, it's fine how it is.